src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs

```diff
@@ -89,15 +90,28 @@ public Task<CompletionSet> TargetLocationSearch(string searchText, int caretPosi
 
             foreach (Tuple<string, string> completion in completions)
             {
-                CompletionItem completionItem = new CompletionItem
+                bool addItem = false;
+                string insertionText = completion.Item2;
+
+                if (insertionText.IndexOf(searchText, StringComparison.OrdinalIgnoreCase) > -1)
+                {
+                    addItem = true;
+                }
+
+                if (addItem)
                 {
-                    DisplayText = completion.Item1,
-                    InsertionText = completion.Item2,
-                };
+                    CompletionItem completionItem = new CompletionItem
+                    {
+                        DisplayText = completion.Item1,
+                        InsertionText = completion.Item2,
+                    };
 
-                completionItems.Add(completionItem);
+                    completionItems.Add(completionItem);
+                }
             }
 
+            completionItems = completionItems.OrderBy(m => m.InsertionText.ToLower().IndexOf(searchText.ToLower())).ToList();
```

Comment:
>m.InsertionText.ToLower().IndexOf(searchText.ToLower()) [](start = 59, length = 55)

Is this the same as m.InsertionText.IndexOf(searchText, StringComparison.OrdinalIgnoreCase)